tracer/src/Datadog.Trace/AppSec/ObjectExtractor.cs

```diff
@@ -188,5 +243,14 @@ private static List<object> ExtractListOrArray(object value, int depth, HashSet<
 
             return items;
         }
+
+        private class FieldExtractor
+        {
+#pragma warning disable CS0649, SA1401
+            public string Name;
+            public Type Type;
+            public Func<object, object> Accessor;
+#pragma warning restore CS0649, SA1401
+        }
```

Comment:
You could just use properties and not disable all those warnings 😛 
```suggestion
        private class FieldExtractor
        {
            public string Name { get; set; }

            public Type Type { get; set; }

            public Func<object, object> Accessor  { get; set; }
        }
```

Reply:
I thought using properties would be slower, but I checked the numbers and it doesn't seem to make a difference, so made the change in 4f7fa5c48